Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -2402,6 +2415,19 @@ void WebProcessProxy::systemBeep()
     PAL::systemBeep();
 }
 
+void WebProcessProxy::sendStorageAccessUserAgentStringQuirksData()
```

Comment:
Should this be in WebPageProxy instead? We send the link decoration filtering data from there.

Reply:
We could. I'm using the WebProcess because these quirks aren't specific to any web page. I guess that's true for LDF data, too, though. I started by following how we do it for link decoration filtering, but it seemed more complicated with multiple checks in `decidePolicyForNavigationAction`. But I guess WebProcessProxy has its own complexity due to `shouldSendPendingMessage`.